_data/nathanlloydw/nathanlloydw.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Nathan Williams
+institution: Teesside University
+profile_pic: nathanlloydw.jpg
+quote: "Code is like humor. When you have to explain it, it’s bad."
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format